pandas/stats/moments.py

```diff
@@ -1036,3 +1036,72 @@ def expanding_apply(arg, func, min_periods=1, freq=None,
     window = max(len(arg), min_periods) if min_periods else len(arg)
     return rolling_apply(arg, window, func, min_periods=min_periods, freq=freq,
                          args=args, kwargs=kwargs)
+
+#----------------------------------------------------------------------
+# Add all the methods to DataFrame and Series
+import sys
+thismodule = sys.modules[__name__]
+
```

Comment:
you are creating lots of machinery that can simply use ``PandasDelegate``, see ``pandas/tseries.common.py`` for an example